vscode/src/webview/webview.tsx

```diff
@@ -6,13 +6,19 @@
 const vscodeApi = acquireVsCodeApi();
 
 import { render } from "preact";
-import { EstimatesPanel, Histogram, type ReData } from "qsharp-lang/ux";
+import {
+  CircuitPanel,
+  EstimatesPanel,
+  Histogram,
+  type ReData,
+} from "qsharp-lang/ux";
 import { HelpPage } from "./help";
 
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore - there are no types for this
 import mk from "@vscode/markdown-it-katex";
 import markdownIt from "markdown-it";
+import { CircuitData } from "qsharp-lang/ux/circuit";
```

Comment:
I think today we import everything from either `qsharp-lang` or `qsharp-lang/ux`. If you reexport this type at the top level as you do for Circuit and CircuitPanel already, then just add this import with those on line 9.